dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/layerparam/initializer/ConvolutionalLayerParameterInitializer.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2016 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.dolphin.dnn.layerparam.initializer;
+
+import edu.snu.dolphin.dnn.blas.Matrix;
+import edu.snu.dolphin.dnn.blas.MatrixFactory;
+import edu.snu.dolphin.dnn.conf.LayerConfigurationParameters;
+import edu.snu.dolphin.dnn.layers.LayerParameter;
+import org.apache.reef.tang.annotations.Parameter;
+
+import javax.inject.Inject;
+
+import static edu.snu.dolphin.dnn.util.NeuralNetworkUtils.shapeFromString;
+
+/**
+ * Convolutional Layer parameter initializer.
+ *
+ * initializes the weight matrix with pseudo random normal distributed value with mean 0 and given standard deviation.
+ * initializes the bias vector with the given value.
+ * includes function that computes the output shape.
```

Comment:
Could you add subjects for these sentence?
If you're okay, please add subjects in `FullyConnectedLayerParameterInitializer`'s description, too.